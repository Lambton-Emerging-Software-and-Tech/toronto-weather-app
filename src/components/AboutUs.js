import React, { Component } from "react";

export default class AboutUs extends Component {
  render() {
    return (
      <div className="w-full h-full flex flex-center justify-center mt-10 sm:mt-10">
        <div className="flex flex-col align-center justify-center">
          <div className="text-4xl mb-5">Team Members</div>
          <div>Milan Lakhani</div>
          <div>Jaimin Patel</div>
          <div>Nihar Dhanani</div>
          <div>Nehalkumar Jesdadia</div>
          <div>Gurmanjot Chemma</div>
        </div>
      </div>
    );
  }
}
