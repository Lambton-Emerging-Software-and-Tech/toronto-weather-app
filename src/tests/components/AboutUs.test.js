import React from 'react';
import { shallow } from 'enzyme';
import AboutUs from '../../components/AboutUs';

test('should test AboutUs component', () => {
 const wrapper = shallow(<AboutUs />);
 expect(wrapper).toMatchSnapshot();
});