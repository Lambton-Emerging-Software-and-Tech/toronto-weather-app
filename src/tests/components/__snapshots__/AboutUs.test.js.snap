// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test AboutUs component 1`] = `
<div>
  This is About Us
</div>
`;
